feature/bgp/policybase/ate_tests/route_installation_test/route_installation_test.go

```diff
@@ -150,10 +150,14 @@ func bgpCreateNbr(localAs, peerAs uint32, policy string) *telemetry.NetworkInsta
 
 	d := &telemetry.Device{}
 	ni1 := d.GetOrCreateNetworkInstance(*deviations.DefaultNetworkInstance)
-	bgp := ni1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").GetOrCreateBgp()
+	ni_proto := ni1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP")
+	bgp := ni_proto.GetOrCreateBgp()
+	ni_proto.Name = ygot.String("BGP")
+	ni_proto.Identifier = telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP
```

Comment:
This shouldn't be needed. `GetOrCreateProtocol` above calls `NewProtocol` which sets these fields explicitly. Can you confirm why it is added?

Reply:
Not required since it calls `NewProtocol` I'll remove L155+L156.

Reply:
if we don't need L155+L156, what is the diff between old L153  and new L153 + L154? I guess we don't need to change this part...